Formula/lilypond.rb

```diff
@@ -0,0 +1,66 @@
+class Lilypond < Formula
+  desc "Music notation for everyone"
+  homepage "https://lilypond.org"
+  url "https://lilypond.org/download/sources/v2.22/lilypond-2.22.1.tar.gz"
+  sha256 "72ac2d54c310c3141c0b782d4e0bef9002d5519cf46632759b1f03ef6969cc30"
+  license "GPL-3.0-or-later"
+
+  livecheck do
+    url "https://lilypond.org/source.html"
+    regex(/href=.*?lilypond[._-]v?(\d+(?:\.\d+)+)\.t/i)
+  end
+
+  depends_on "bison" => :build # Lilypond requires bison 2.4.1 or above
+  depends_on "fontforge" => :build
+  depends_on "gettext" => :build
+  depends_on "glib" => :build
+  depends_on "perl" => :build
+  depends_on "pkg-config" => :build
+  depends_on "t1utils" => :build
+  depends_on "texinfo" => :build # lilypond requires texinfo 6.1 or above
+  depends_on "texlive" => :build
+
+  depends_on "fontconfig"
+  depends_on "freetype"
+  depends_on "ghostscript"
+  depends_on "guile@2"
+  depends_on "pango"
+  depends_on "python@3.9"
+
+  uses_from_macos "flex" => :build
+
+  def install
+    # needed to solve conflicts with emacs
+    inreplace "config.make.in",
+      %r{^\s*elispdir\s*=\s*\$\(datadir\)/emacs/site-lisp\s*$},
+      "elispdir = $(datadir)/emacs/site-lisp/lilypond"
+
+    system "./configure",
+            "--prefix=#{prefix}",
+            "--with-texgyre-dir=#{Formula["texlive"].opt_libexec}/share/texmf-dist/fonts/opentype/public/tex-gyre",
+            "--disable-documentation"
+    ENV.prepend_path "LTDL_LIBRARY_PATH", Formula["guile@2"].lib
+    system "make"
+    system "make", "install"
+
+    libexec.install bin/"lilypond"
+
+    (bin/"lilypond").write <<~EOS
+      #!/bin/sh
+      GUILE_WARN_DEPRECATED=no LTDL_LIBRARY_PATH="#{Formula["guile@2"].opt_libexec/"lib"}:$LTDL_LIBRARY_PATH" #{opt_libexec}/lilypond "$@"
```

Comment:
`guile@2` doesn’t install a libexec folder. `Formula["guile@2"].opt_libexec/"lib"` (which doesn’t exist) should be `Formula["guile@2"].lib`.

Reply:
Oh, yes, I forgot to change this line, but I'm not sure if this is still necessary as it worked even with the wrong path.